sx-auth.el

```diff
@@ -38,14 +38,16 @@
   "http://vermiculus.github.io/sx.el/auth/auth.htm")
 (defconst sx-auth-client-id
   "3291")
-(defvar sx-auth-access-token
-  nil
+
+(defstash sx-auth-access-token
+    "auth.el" sx nil
```

Comment:
Oh I don't know. I don't think we should change this variable specifically. It just occurred to me we might want to use longer names on future stashes.